db/seeds.rb

```diff
@@ -833,3 +833,7 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   ]
 )
 TransferCreateService.call(transfer)
+
```

Comment:
❓Question: For this file I see some common comments sections, should I add some comments to explain why the users' `invitation_status` is being changed?

Reply:
Can't hurt!